genobuilder.py

```diff
@@ -831,7 +834,26 @@ def locate(sorted_idx, start=None, stop=None):
         params_dict["T5"] = Parameter("T5", 8533, (5001, 10000), inferable=False)
         params_dict["N5"] = Parameter("N5", 71560, (1000, 100000), inferable=True)
 
+    elif args.demographic_model == "bottleneck":
+        # Parameters for exponential model: FIX   BOUNDS      INFERABLE
+        params_dict["N0"] = Parameter("N0", 10000, (100, 30000), inferable=True)
+        params_dict["T1"] = Parameter("T1", 1000, (100, 1500), inferable=False)
+        params_dict["N1"] = Parameter("N1", 1000, (100, 30000), inferable=True)
+        params_dict["T2"] = Parameter("T2", 2000, (1500, 5000), inferable=False)
+        params_dict["N2"] = Parameter("N2", 10000, (100, 30000), inferable=True)
+
+        import msprime
+        demographic_events = [
+            msprime.PopulationParametersChange(time=0, initial_size=params_dict["N0"].val),
+            msprime.PopulationParametersChange(time=params_dict["T1"].val, initial_size=params_dict["N1"].val),
+            msprime.PopulationParametersChange(time=params_dict["T2"].val, initial_size=params_dict["N2"].val),
+        ]
+
+        debugger = msprime.DemographyDebugger(Ne=10000, demographic_events=demographic_events)
+        debugger.print_history()
```

Comment:
I think it's quite fragile to copy the model here and look at the debug output. It would be better if the `bottleneck()` function in demography.py were broken into two functions, one that returns the msprime Demography object (say, `bottleneck_model()`), and then just call this to get the model in the `bottleneck()` function for simulation. The code here in genobuilder.py would then call the `bottleneck_model()` function and we would be sure that the debug output here corresponds exactly with the model being simulated.